### DISCUSSION TITLE
How to set day_offset in UI?

### DISCUSSION THREAD

**Discussion by jaredcat** [mampfes/hacs_waste_collection_schedule#2692] [Q&A] [answered]
Using the UI setup, how can I set a negative `day_offset`?

**Comment by 5ila5** [comment, marked as answer]
Oh, it seems like this option was forgotten when creating the config_flow (GUI setup), probably because it's a newer option and development on the GUI configuration started before day_offset could be set.

I added day_offset to the GUI in #2699 

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA

install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
